community/kernel/src/main/java/org/neo4j/unsafe/impl/batchimport/input/csv/InputEntityDeserializer.java

```diff
@@ -176,6 +186,11 @@ private boolean deserializeNextFromSource() throws IOException
     @Override
     public void close()
     {
+        if ( ignoredColumns > 0 )
+        {
+            System.out.println( String.format( "Warning: Ignored %d extra columns in %s", ignoredColumns,
```

Comment:
Yeah, perhaps gather line numbers (at least the first 10 or something), and some sample data?